internal/processor/rule.go

```diff
@@ -185,6 +188,25 @@ func (p *RuleProcessor) GetRuleByJson(id, ruleJson string) (*api.Rule, error) {
 	return rule, nil
 }
 
+func validateRuleID(id string) error {
+	for _, char := range id {
+		if !isLegalChar(char) {
+			return fmt.Errorf("ruleID:%v invalid", id)
+		}
+	}
+	return nil
+}
+
+func isLegalChar(char rune) bool {
+	if (char >= '0' && char <= '9') ||
```

Comment:
Is Chinese name still supported? Shall we find a common rule which can still tolerate most of the names.